src/commands/register.ts

```diff
@@ -20,6 +21,8 @@ import { IF_EXPRESSION_REGEX } from '../dependency-manager/spec/utils/interpolat
 
 tmp.setGracefulCleanup();
 
+export const EPHEMERAL_DELIMITER = 'architect-ephemeral';
```

Comment:
Should we make it targeted to the environment? Not really a delimiter - more a prefix.
```
export const ENV_TAG_PREFIX = 'architect.environment.'
```

Reply:
Yeah I like that